coffeecard/CoffeeCard.WebApi/Controllers/v2/AccountController.cs

```diff
@@ -0,0 +1,52 @@
+using CoffeeCard.Common.Errors;
+using CoffeeCard.Library.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CoffeeCard.WebApi.Controllers.v2
+{
+    /// <summary>
+    /// Controller for creating and managing user accounts
+    /// </summary>
+    [ApiController]
+    [ApiVersion("2")]
+    [Route("api/v{version:apiVersion}/[controller]")]
```

Comment:
We want to force the url for this controller to be in lowercase
```suggestion
    [Route("api/v{version:apiVersion}/account")]
```